src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -161,13 +161,21 @@ angular.module('Module.emailpro.controllers')
   });
 
 angular.module('Module.emailpro.controllers')
-  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro) => {
+  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro, User) => {
     // default values
     $scope.accountsToAdd = {
       duration: '12',
       accountsNumber: 1,
     };
 
+    User.getUser()
+      .then((currentUser) => {
+          this.isGermanSubsidiary = currentUser.ovhSubsidiary === "DE";
+      })
+      .catch(() => {
+          this.isGermanSubsidiary = false;
```

Comment:
If you can't retrieve the User, maybe you should warn the user that there is an issue

Reply:
This is done :)